src/DiffOp/Diff/Diff.php

```diff
@@ -64,7 +64,7 @@ public function __construct( array $operations = [], $isAssociative = null ) {
 		parent::__construct( [] );
 
 		foreach ( $operations as $offset => $operation ) {
-			if ( !( $operation instanceof DiffOp ) ) {
+			if ( !( $operation instanceof DiffOpInterface) ) {
```

Comment:
Missing a space inside the parentheses (after `DiffOpInterface`) and in a few other similar cases (also in other files). Our coding style mandates spaces inside parentheses, for both function calls, like `foobar( 42 );`, and for conditionals like here.

(Unrelated to the PR: It seems that phpcs is not configured to check for this here, although I think it should.)